### ISSUE TITLE
How can I setup default download directory?

### ISSUE THREAD

**Issue by idkang72** [inbasic/iaextractor#212]
Hi, I'm your mozilla addon user.
First, thank you for your addon.
I'm using it happily.

But I want one thing for more happy life.
In previos version, I could change default download directory.
but, I cannot find setting in latest version.

How can I setup default download directory?
Or Is it possible to remember last download directory?

**Comment by tjhleeds** [comment]
I agree, it would be really helpful to be able to set the default download directory.
